### PR TITLE
Add Notifications Adjustments

### PR DESCRIPTION
**Pull request by igaponov**

### CODE REVIEW

File: backend/src/Civix/CoreBundle/Service/PushSender.php [hunk 1]
```diff
@@ -173,20 +231,34 @@ public function sendSocialActivity($id)
...
             }
         }
     }
     
-    public function send(User $recipient, $messageBody, $type, $entityData = null)
+    public function send(User $recipient, $title, $message, $type, $entityData = null, $image = null)
     {
         $endpoints = $this->entityManager->getRepository(AbstractEndpoint::class)->findByUser($recipient);
+        if (empty($image)) {
+            $image = 'www/images/notification_image.jpg';
```
Comment (austinpapp) [inline backend/src/Civix/CoreBundle/Service/PushSender.php]: thinking we make this a const ?

**Comment by austinpapp** [comment]
@igaponov i'm ok with the imgix lib. however, is that avatar filename the full s3 path to reference?

**Comment by jterps08** [comment]
@austinpapp the imgix processing URL should be different depending on whether or not we're in the production, staging, or development environments because the imgix processor is linked directly to the S3 bucket... http://powerline-prod.imgix.net/
http://powerline-staging.imgix.net/
http://powerline-dev.imgix.net/

**Comment by austinpapp** [comment]
@jterps08 ah! perfect. thanks.

File: backend/src/Civix/CoreBundle/Service/PushSender.php [hunk 2]
```diff
@@ -195,10 +267,18 @@ public function send(User $recipient, $messageBody, $type, $entityData = null)
...
 
         return $text;
     }
+
+    private function getLinkByFilename($fileName)
+    {
+        return $this->urlBuilder->createURL(
+            $fileName,
+            array("dpr" => 0.75, "w" => 320, "h" => 400)
+        );
```
Comment (jterps08) [inline backend/src/Civix/CoreBundle/Service/PushSender.php]: @austinpapp @igaponov I'm not sure if these are the correct dimensions. I was just using the 320x400 as an example for how to use the imgix URL... I'm not sure what the best dimensions are to use for a notification image. I assume, small...

Comment (austinpapp) [inline backend/src/Civix/CoreBundle/Service/PushSender.php]: i'm not sure it matters much as long as it follows the 2:3:4:6:8 scaling ratio. however, this does not need to be high density images. lo-fi basic works in my opinion.

@igaponov we may want to make these `const` so we can easily change if necessary.

File: backend/src/Civix/CoreBundle/Service/PushSender.php (hunk 1 above)
Comment (igaponov) [inline backend/src/Civix/CoreBundle/Service/PushSender.php]: Done

File: backend/src/Civix/CoreBundle/Service/PushSender.php (hunk 2 above)
Comment (igaponov) [inline backend/src/Civix/CoreBundle/Service/PushSender.php]: Done

**Pull request merged**